packages/@aws-cdk/aws-ecr/lib/repository.ts

```diff
@@ -327,6 +328,27 @@ export interface RepositoryProps {
    */
   readonly repositoryName?: string;
 
+  /**
+   * The kind of server-side encryption to apply to this repository.
+   *
+   * If you choose KMS, you can specify a KMS key via `encryptionKey`. If
+   * encryption key is not specified, an AWS managed KMS key is used.
```

Comment:
Use the exact name of the property: 

```suggestion
   * encryptionKey is not specified, an AWS managed KMS key is used.
```